src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GoToGCPConsoleStep.ts

```diff
@@ -0,0 +1,39 @@
+import { h } from 'react-hyperscript-helpers'
+import { ButtonOutline } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import Events from 'src/libs/events'
+import * as Utils from 'src/libs/utils'
+import { Step } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/Step'
+import { StepFieldLegend, StepFields } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepFields'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+interface GoToGCPConsoleStepProps {
+  isActive: boolean
+  stepFinished: () => void
+}
+
+export const GoToGCPConsoleStep = ({ isActive, ...props }: GoToGCPConsoleStepProps) => {
+  return h(Step, { isActive }, [
+    StepHeader({ title: 'STEP 1' }),
+    h(StepFields, [
+      h(StepFieldLegend, ['Go to the Google Cloud Platform Billing Console and sign-in with the same user you use to login to Terra.']),
+      h(ButtonOutline, {
+        disabled: false,
+        href: 'https://console.cloud.google.com',
+        ...Utils.newTabLinkProps,
+        onClick: () => {
+          Ajax().Metrics.captureEvent(Events.billingGCPCreationStep1)
+          // FIXME: this seems wrong
```

Comment:
I think this might be tied into to the fact that we didn't want to force people to click on the button, and we want people to be able to interact with the previous controls even when they are on a different step. It's all pretty complicated though.